src/Context.hpp

```diff
@@ -107,6 +107,10 @@ class Context : NonCopyable
   // Headers (or directories with headers) to ignore in manifest mode.
   std::vector<std::string> ignore_header_paths;
 
+  // Options to completely ignore on the command line. They will not be passed
+  // to either the preprocessor or the compiler.
+  std::vector<std::string> ignore_options;
```

Comment:
' They will not be passed to either the preprocessor or the compiler' - really? Ccache shall ignore them, but still pass them on unmodified?! Or did I get the idea here wrong?

Reply:
That comment was from before I fully understood what I was doing, and I missed changing it accordingly once I did. I’ll fix it.